wallet-connect-test-bench/front-end/package.json

```diff
@@ -22,7 +24,7 @@
   "resolutions": {
     "@concordium/wallet-connectors": "../../deps/concordium-dapp-libraries/packages/wallet-connectors",
     "@concordium/react-components": "../../deps/concordium-dapp-libraries/packages/react-components",
-    "@concordium/web-sdk": "^6.0.0"
+    "@concordium/web-sdk": "^7.0.0"
```

Comment:
I guess this is needed as we have the dapp libraries as a local dependency?

Reply:
Yes, it was needed at some point when a version mismatch of this dependency here in the dapp libraries vs the dependency used in the wallet-connect-bench caused some problems.

I haven't investigated if we still need the resolution but since adding the resolution there were no further version mismatch problems.